doc/tutorials/basic/pn_rx.md

```diff
@@ -0,0 +1,320 @@
+# Reactive Expressions
+
+In this section you will learn about `pn.rx`. `pn.rx` extends the concepts from `pn.bind` that your learned in the previous section.
+
+:::{note}
+You might feel some repetition from the previous section on `pn.bind`. We do this on purpose to enable you to compare and contrast. `pn.rx` is the new, and much more flexibly big brother of `pn.bind`. But `pn.bind` has been the core api in Panel for a long time, so you will meet it across our documentation and community sites, and thus its very important to learn.
+
+`pn.rx` will enable you to build more complicated applications using a more flexible and maintainable architecture.
+:::
+
+## Embrace `pn.rx`
+
+```{pyodide}
+import panel as pn
+
+pn.extension()
+
+def calculate_power_generation(wind_speed, efficiency):
+    power_generation = wind_speed * efficiency
+    return f"Wind Speed: {wind_speed} m/s, Efficiency: {efficiency}, Power Generation: {power_generation:.1f} kW"
+
+wind_speed = pn.widgets.FloatSlider(
+    value=5, start=0, end=20, step=1, name="Wind Speed (m/s)"
+)
+
+efficiency = 0.3
+
+calculate_power_fn = pn.rx(calculate_power_generation)(wind_speed=wind_speed, efficiency=efficiency)
+
+pn.Column(
+    calculate_power_fn
+).servable()
+```
+
+You will notice how adding `calculate_power_fn` to the `Column` displays both the widget and the bound function.
+
+To separate the widget and bound function we use the `calculate_power_fn` as a reference to a Pane:
+
+```{pyodide}
+import panel as pn
+
+pn.extension()
+
+def calculate_power_generation(wind_speed, efficiency):
+    power_generation = wind_speed * efficiency
+    return f"Wind Speed: {wind_speed} m/s, Efficiency: {efficiency}, Power Generation: {power_generation:.1f} kW"
+
+wind_speed = pn.widgets.FloatSlider(
+    value=5, start=0, end=20, step=1, name="Wind Speed (m/s)"
+)
+
+efficiency = 0.3
+
+calculate_power_fn = pn.rx(calculate_power_generation)(wind_speed=wind_speed, efficiency=efficiency)
+
+pn.Column(
+    wind_speed, pn.pane.Markdown(calculate_power_fn)
+).servable()
+```
+
+:::{note}
+If you want all your reactive expressions displayed without widgets you can disable the widgets by setting
+
+```python
+pn.ReactiveExpr.show_widgets=False
```

Comment:
Please provide fix to https://github.com/holoviz/panel/issues/6428 or workaround Philipp.